### DISCUSSION TITLE
Flows Folders

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#14610] [Feature Requests]
Would be nice to group Flows in folders, similar to folders in "Data Model" would certainly help organize things vs long titles I'm currently using to identify which flow is which.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
